### ISSUE TITLE
User Name ALIASES for different spaces channels rooms etc

### ISSUE THREAD

**Issue by jittygitty** [vector-im/element-android#6058]
#### What would you like to do?
Some people may know me differently and be used to calling me by different names. 
This feature is just for simple ALIASES to the same login username/profile, so that different people/rooms/spaces can see your username by specifically the alias they know you by...

#### Why would you like to do it?
You tell me.

#### How would you like to achieve it?
By asking?

**Comment by bbhtt** [comment]
There is a `/myroomnick` command that can set a per-room nick. So you can have different display names for different rooms and a global one. Is that what you are looking for?

Note that the per-room nick can sometime leak into the global one, this bug is tracked as part of https://github.com/vector-im/element-android/issues/1715.

**Comment by jittygitty** [comment]
@bbhtt Yes actually that's useful, I didn't know of the /myroomnick command, thanks! 

Now my other request at https://github.com/vector-im/element-android/issues/6053  was mainly for just a "client" side thing to set anyone's name to whatever I want (would only affect my view/UI), I wouldn't care if we don't tell the server anything about that really, to be just a UI thing for my client app since that might be simplest to do.

**Comment by bbhtt** [comment]
Hi, I think you should make a separate issue for that particular purpose. This was requested but it'd require a MSC for that, see https://github.com/matrix-org/matrix-spec/issues/188
